exercises/ocr-numbers/ocr_numbers_test.exs

```diff
@@ -0,0 +1,254 @@
+if !System.get_env("EXERCISM_TEST_EXAMPLES") do
+  Code.load_file("ocr_numbers.exs", __DIR__)
+end
+
+ExUnit.start
+ExUnit.configure exclude: :pending, trace: true
+
+defmodule OCRNumbersTest do
+  use ExUnit.Case
+
+  # @tag :pending
+  test "Recognizes 0" do
+    number = OCRNumbers.convert(
+      [
+        " _ ",
+        "| |",
+        "|_|",
+        "   "
+      ]
+    )
+    assert number == "0"
+  end
+
+  @tag :pending
+  test "Recognizes 1" do
+    number = OCRNumbers.convert(
+      [
+        "   ",
+        "  |",
+        "  |",
+        "   "
+      ]
+    )
+
+    assert number == "1"
+  end
+
+  @tag :pending
+  test "Unreadable but correctly sized inputs return ?" do
+    number = OCRNumbers.convert(
+      [
+        "   ",
+        "  _",
+        "  |",
+        "   "
+      ]
+    )
+
+    assert number == "?"
```

Comment:
I debated this for a while since the metadata about this problem wanted `-1` as an error condition, but I am glad that the Elixir-y way is preferred. I will make this change shortly.